docs/software/switch-config.md

```diff
@@ -8,6 +8,46 @@ description: To optimize the performance of ExpressLRS, it has different Switch
 !!! warning "WARNING" 
     **Put your arm switch on AUX1**, and set it as **~1000 is disarmed, ~2000 is armed**.
 
+# Summary of Switch Configs
+
+This table summarizes the switch configuration modes and the available switch positions / resolution and update frequency on each channel or flight controller auxiliary channel (Aux X).
+
+| Channel | Flight  <br>Controller | Hybrid | Wide  <br>Hybrid | Full Res  <br>8 | Full Res  <br>16 Half Rate | Full Res  <br>12 Mixed |
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| 1   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 2   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 3   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| 4   | Sticks | **Full Res** | **Full Res** | **Full Res** | Full Res<sup>H</sup> | **Full Res** |
+| **5<sup>+</sup>**   | Aux 1 | **2-pos** | **2-pos** | **2-pos** | **2-pos** | **2-pos** |
+| 6   | Aux 2 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | *Full Res*<sup>RR</sup> | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 7   | Aux 3 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | *Full Res*<sup>RR</sup> | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 8   | Aux 4 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | *Full Res*<sup>RR</sup> | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 9   | Aux 5 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | *Full Res*<sup>RR</sup> | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 10  | Aux 6 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | - | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 11  | Aux 7 | *6-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | - | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 12  | Aux 8 | *16-pos*<sup>RR</sup> | *64/128<br>-pos*<sup>RR</sup> | - | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 13  | Aux 9 | - | - | - | Full Res<sup>H</sup> | Full Res<sup>H</sup> |
+| 14  | Aux 10 | - | - | - | Full Res<sup>H</sup> | - |
+| 15  | Aux 11 | - | - | - | Full Res<sup>H</sup> | - |
+| 16  | Aux 12 | - | - | - | Full Res<sup>H</sup> | - |
+ 
+**Switch Position / Channel Resolution**<br>
+**Full Res** = 10-bit or 1024 positions for full resolution for a PWM between 988 to 2012<br>
+**64 / 128-pos** = 6/7-bit or 64/128 positions (approx 12.5% the resolution of 10-bit). The bit depth is dependent on the selected telemetry ratio<br>

```

Comment:
Might want to simplify this instead of being nebulous. For 1:2, 1:4 telemetry ratio they are 6-bit/64 pos. For all other ratios, they are 7-bit/128 pos.

Reply:
Nice, I like the way it puts the positions next to the bit length.

Reply:
I think I got it good enough to be understandable.